configure.ac

```diff
@@ -1683,11 +1683,7 @@ if test -n "$OCTAVE"; then
    for var in OCTLIBDIR; do
      OCTAVE_LDFLAGS="${OCTAVE_LDFLAGS} "-L`${mkoctfile} -p ${var}`
    done
-   for var in RDYNAMIC_FLAG RLD_FLAG OCTAVE_LIBS LIBS; do
-     # RLD_FLAG gives "mkoctfile: unknown variable 'RLD_FLAG'" on stderr
-     # with Octave 7.3 so just discard stderr here.  Apparently RLD_FLAG has
-     # reported an empty value since somewhere between 3.4.3 and 3.6.1 so
-     # can be removed below once we require Octave 4.
+   for var in RDYNAMIC_FLAG OCTAVE_LIBS; do
```

Comment:
This also drops `RLD_FLAG` which seems like it probably means dropping support for Octave 4.  Maybe that's fine, but even if it is it seems unrelated to the described purpose of your proposed change so would be better proposed separately (and ought to include an update to the manual to note the oldest version now supported).

Reply:
> probably means dropping support for Octave 4

I think I meant "probably means requiring at least Octave 4".  I don't know much about Octave and what versions are still likely in use, but the oldest we CI test is 5.2 and the current version is 9 so that seems likely to be an entirely reasonable change, but my points about not lumping it in with this change and the docs needing updating still stand.